### PR TITLE
Updated the points Criteria

### LINKED ISSUE #112: Update the points criteria
Update the points criteria here:

Screenshot from 2019-11-10 14-14-15

New points criteria is as follows:

Extra: 5 points
Very Easy: 10 points
Easy: 15 points
Medium: 25 points
Hard: 30 points
Pro: 50 points
Codeburst: 100 points

### PR DESCRIPTION
**Pull request by yk1ax**
Closes #112 
Updated the points criteria as mentioned in the issue #112

### CODE REVIEW

**Comment by yk1ax** [comment]
@blurry-x-face you can review the PR

**Comment by dojutsu-user** [comment]
Please add one more label, `codeburst` - 100 points

**Comment by yk1ax** [comment]
@dojutsu-user what is this codeburst label

**Comment by dojutsu-user** [comment]
It is for `100 points`.
Currently there are no issue which have this label on it, but we plan to open it soon. :smiley:

**Comment by yk1ax** [comment]
What are the criteria for this label ???

**Comment by dojutsu-user** [comment]
@yk1ax There's no criteria... we will be opening issues of that that label soon.

**Comment by yk1ax** [comment]
@dojutsu-user review this PR

**Comment by dojutsu-user** [comment]
@yk1ax 
Please add `codeburst` also.

**Comment by yk1ax** [comment]
@dojutsu-user I can't see any option to add label
I think only Owner or Collaborators can add labels

**Comment by yk1ax** [comment]
<code>codeburst</codeburst>

**Comment by yk1ax** [comment]
`codeburst`

File: index.html [hunk 1]
```diff
@@ -161,11 +161,12 @@ <h2 class="title-hl wow fadeInLeft" data-wow-delay="0.3s">
...
-                    &nbsp;&nbsp;Easy: 15<br />
-                    &nbsp;&nbsp;Medium: 25<br />
-                    &nbsp;&nbsp;Hard: 30<br />
-                    &nbsp;&nbsp;Pro: 50
+                    &nbsp;&nbsp;Extra: 5 points<br />
+                    &nbsp;&nbsp;Very Easy: 10 points<br />
+                    &nbsp;&nbsp;Easy: 15 points<br />
+                    &nbsp;&nbsp;Medium: 25 points<br />
+                    &nbsp;&nbsp;Hard: 30 points<br />
+                    &nbsp;&nbsp;Pro: 50 points
```
Comment (dojutsu-user) [inline index.html]:
```suggestion
                    &nbsp;&nbsp;Pro: 50 points
                    &nbsp;&nbsp;Codeburst: 100 points
```

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

**Review by dojutsu-user — APPROVED** [review]

**Pull request merged**